src/three-components/CachingGLTFLoader.ts

```diff
@@ -207,4 +214,13 @@ export class CachingGLTFLoader {
 
     return model as CacheRetainedScene | null;
   }
+
+  async[$load](url: string, progressCallback: ProgressCallback = () => {}) {
```

Comment:
This looks almost like your loadWithLoader helper function; what's the difference?

Reply:
No difference, effectively. The fact that the loader didn't have a self-owned method for loading seemed smelly to me as I was going through, so I cleaned it up. I could take the refactor or leave it though.